diffpy/nmf_mapping/nmf_mapping/main.py

```diff
@@ -34,11 +34,14 @@ def main(args=None):
     parser = ArgumentParser(prog="nmf_mapping", description=_BANNER, formatter_class=RawTextHelpFormatter)
 
     def tup(s):
```

Comment:
I think there is probably a better method for handling this with argparse.  Also, it seems that the issue here would be that after parsing the string, they should be two integers, but why not just use the argparse ways of handling this?